docs/contributing/browser-support.md

```diff
@@ -0,0 +1,215 @@
+- Optional enhancements - which do not prevent users from using the component if they’re absent or working more simply. For example, the ability to search within sections of an Accordion without opening them
+
+## Necessary enhancements
+
+We will make sure necessary enhancements are available in all browsers running our JavaScript enhancements (Grades A, B and C). Wherever possible, we aim to provide the same baseline experience across these browsers.
+
+## Optional enhancements
+
+For all grade A and grade B browsers, we aim to provide the same baseline experience. One exception is enhancements that require vendor specific features. For example, `hidden="until-found"` which is only implemented in Chrome at the time of writing.
+
+For grade C browsers, we aim to support optional enhancements as much as possible. However, we might sometimes need to adjust the component’s behaviour, while still allowing the users to complete their task. This might include:
+
+- falling back to the no-JavaScript experience
+- disabling parts of the enhancement
+- doing a simpler alternative
+
+When deciding how much support we provide to grade C browsers using Javascript enhancements, we will take into account:
+
+- what impact removing the enhancement will have on users and service teams
+- whether there is wide usage of browser features used by our Javascript enhancements - for example, we'll provide support if a large number of grade C browsers have features for specific enhancements
+- whether the enhancement is important for certain types of user, for example, if the feature is an accessibility enhancement that benefits screen reader users
+- the overhead of polyfilling and/or transpiling, in terms of the amount of extra JavaScript users need to download, or the effect it has on loading and runtime performance
+- the engineering effort required, for the initial work and to maintain and support the enhancement or polyfill over time
+- our confidence in our understanding of all of the previous points in this list
+
+Where behaviour might differ in older browsers, we will document those differences in the Design System. We will also communicate any changes in our release notes.
+
+## How we support browsers
+
+Supporting a browser involves more than considering whether JavaScript enhancements run in them. It includes:
+
+- the level of testing browsers will receive, such as
+- manual testing during development and prior to releases
+- using our automated test suites
+- the support for JavaScript
+- browsers downloading and parsing the script without errors
+- necessary enhancements being available to users
+- optional enhancements being available to users
+- bug prioritisation
+
+This table shows the level of support for the 3 grades of browsers (A, B and C) which run our JavaScript.
+
+We’ve excluded Grade X browsers from this table as we’re not supporting enhancements for them and the prioritisation for bugs is ‘will not fix’.
+
+<table>
+    <tbody>
```

Comment:
The table looks great on wider screens, but it's super squished on mobiles 😣

Think we can find an alternative?

Reply:
We disable Prettier `embeddedLanguageFormatting` but probably stick to 2-spaces?

```suggestion
<table>
  <tbody>
```

Reply:
I can move everything out into bullet points for each grade?

Reply:
Suggest we leave the table in for this pre-release, then think about another solution for the next.